src/network/network.module.ts

```diff
@@ -0,0 +1,15 @@
+/* istanbul ignore file */
+
+import { Module } from '@nestjs/common';
+
+import { NetworkController } from '~/network/network.controller';
+import { NetworkService } from '~/network/network.service';
+import { PolymeshModule } from '~/polymesh/polymesh.module';
+
+@Module({
```

Comment:
I know not many modules have them, but can you add a short TSDoc comment here?
```ts
/**
* responsible for the exposing the SDK network namespace
*/
```
Then it gives a nicer hover over card when its being imported into App module instead of `import NetworkModule`